src/libs/Navigation/linkingConfig/config.ts

```diff
@@ -73,17 +73,9 @@ const config: LinkingOptions<RootStackParamList>['config'] = {
         [SCREENS.NOT_FOUND]: '*',
         [NAVIGATORS.LEFT_MODAL_NAVIGATOR]: {
             screens: {
-                [SCREENS.LEFT_MODAL.CHAT_FINDER]: {
-                    screens: {
-                        [SCREENS.CHAT_FINDER_ROOT]: ROUTES.CHAT_FINDER,
```

Comment:
I think we also need to remove `CHAT_FINDER_ROOT` from `SCREENS.ts` as there is no usage for it now